static/panes/executor.js

```diff
@@ -158,21 +158,23 @@ Executor.prototype.undefer = function () {
 };
 
 Executor.prototype.resize = function () {
-    var topBarHeight = utils.updateAndCalcTopBarHeight(this.domRoot, $(this.topBar[0]), this.hideable);
+    _.defer(function (self) {
```

Comment:
(aside!) We really should start using our transpilation options to open the door to `() => {}` style functions